src/Illuminate/Foundation/Bootstrap/ConfigureLogging.php

```diff
@@ -28,13 +28,6 @@ public function bootstrap(Application $app)
         } else {
             $this->configureHandlers($app, $log);
         }
-
-        // Next, we will bind a Closure that resolves the PSR logger implementation
-        // as this will grant us the ability to be interoperable with many other
-        // libraries which are able to utilize the PSR standardized interface.
-        $app->bind('Psr\Log\LoggerInterface', function ($app) {
```

Comment:
`Psr\Log\LoggerInterfaced` is already aliased in the `Container` : https://github.com/laravel/framework/blob/5.1/src/Illuminate/Foundation/Application.php#L1048